evm/test/forks/HubMainnetFork.t.sol

```diff
@@ -0,0 +1,373 @@
+    values[0] = 0;
+    calldatas = new bytes[](1);
+    calldatas[0] = abi.encodeWithSignature("approve(address,uint256)", address(gov), 0);
+    descriptionHash = keccak256(bytes(_description));
+  }
+
+  // Executes gov.propose with the given parameters after pranking as _proposer.
+  function _proposeFrom(
+    address _proposer,
+    address[] memory _targets,
+    uint256[] memory _values,
+    bytes[] memory _calldatas,
+    string memory _description
+  ) internal returns (uint256 proposalId) {
+    vm.prank(_proposer);
+    proposalId = gov.propose(_targets, _values, _calldatas, _description);
+    assertTrue(proposalId != 0, "Proposal ID is zero");
+  }
+
+  // --- Setup ---
+
+  function setUp() public {
+    // Create a fork of mainnet
+    ethereumForkId = vm.createSelectFork(ETHEREUM_RPC_URL);
+
+    // Load contract instances from known addresses
+    timelock = TimelockController(payable(TIMELOCK_ADDR));
+    gov = HubGovernor(payable(GOV_ADDR));
+    extender = HubProposalExtender(EXTENDER_ADDR);
+    hubVotePool = HubVotePool(HUB_VOTE_POOL_ADDR);
+    hubProposalMetadata = HubProposalMetadata(HUB_METADATA_ADDR);
+    hubMessageDispatcher = HubMessageDispatcher(HUB_MSG_DISPATCHER_ADDR);
+    hubEvmSpokeAggregateProposer = HubEvmSpokeAggregateProposer(HUB_EVM_AGG_PROPOSER_ADDR);
+    hubSolanaMessageDispatcher = HubSolanaMessageDispatcher(HUB_SOLANA_DISPATCHER_ADDR);
+    hubSolanaSpokeVoteDecoder = HubSolanaSpokeVoteDecoder(HUB_SOLANA_VOTE_DECODER_ADDR);
+    // TODO: Replace with actual WToken address for production verification
+    wToken = ERC20Votes(TEST_WTOKEN_ADDR);
+  }
+
+  // --- Parameter Verification Tests ---
+
+  function test_VerifyTimelockParams() public view {
+    assertEq(timelock.getMinDelay(), EXPECTED_MIN_DELAY, "Timelock minDelay mismatch");
+  }
+
+  function test_VerifyGovernorParams() public view {
+    assertEq(gov.name(), EXPECTED_GOV_NAME, "Governor name mismatch");
+    assertEq(address(gov.token()), TEST_WTOKEN_ADDR, "Governor token mismatch");
+    assertEq(address(gov.timelock()), TIMELOCK_ADDR, "Governor timelock mismatch");
+    assertEq(gov.votingDelay(), EXPECTED_VOTING_DELAY, "Governor votingDelay mismatch");
+    assertEq(gov.votingPeriod(), EXPECTED_VOTING_PERIOD, "Governor votingPeriod mismatch");
+    assertEq(gov.proposalThreshold(), EXPECTED_PROPOSAL_THRESHOLD, "Governor proposalThreshold mismatch");
+    assertEq(gov.quorum(block.timestamp), EXPECTED_QUORUM, "Governor initialQuorum mismatch");
+    assertEq(address(gov.hubVotePool(uint96(block.timestamp))), HUB_VOTE_POOL_ADDR, "Governor hubVotePool mismatch");
+    assertEq(address(gov.HUB_PROPOSAL_EXTENDER()), EXTENDER_ADDR, "Governor governorProposalExtender mismatch");
+    assertEq(
+      gov.getVoteWeightWindowLength(uint96(block.timestamp)),
+      EXPECTED_VOTE_WEIGHT_WINDOW,
+      "Governor voteWeightWindow mismatch"
+    );
+  }
+
+  function test_VerifyExtenderParams() public view {
+    // Admin check is in Roles test
```

Comment:
I am seeing this comments, are they necessary?

Reply:
https://github.com/wormhole-foundation/multigov/pull/268/commits/da81d781a2c28f3db1480414ca8b434f0f2d8056

Reply:
removed!